test/docs.jl

```diff
@@ -21,206 +21,3 @@ end
 
 @test (@doc ModuleMacroDoc)    == "I am a module"
 @test (@doc ModuleMacroDoc.@m) == ["I am a macro"]
-
-# apropos function testing
```

Comment:
If you're removing deprecated code you sorta have to get rid of the tests for that code, too.